### PR TITLE
update ce_sflow to fix bugs

### PR DESCRIPTION
**Pull request by yanzhangi**
##### SUMMARY

Optimize parameters for different configurations

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

lib/ansible/modules/network/cloudengine/ce_sflow.py
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/modules/network/cloudengine/ce_sflow.py [hunk 1]
```diff
@@ -127,6 +129,7 @@
               the switch automatically restores it to 65535.
               The value is an integer or 'all'.
         version_added: "2.10"
+        remove_version: "2.13"
```
Comment (yuandongx) [inline lib/ansible/modules/network/cloudengine/ce_sflow.py]: removed_in_version

Comment (yuandongx) [reply lib/ansible/modules/network/cloudengine/ce_sflow.py]: Also in argument_spec , i.e. " server_url=dict(type='str', required=True, removed_in_version="2.10")"

**Review by yuandongx — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/network/cloudengine/ce_sflow.py [hunk 2]
```diff
@@ -109,6 +109,7 @@
             - Specifies the rate of sFlow packets sent from a card to the control plane.
               The value is an integer that ranges from 100 to 1500, in pps.
         version_added: "2.10"
+        remove_version: "2.13"
```
Comment (yuandongx) [inline lib/ansible/modules/network/cloudengine/ce_sflow.py]:
```suggestion
        removed_in_version: "2.13"
```

**Review by yuandongx — CHANGES_REQUESTED** [review]

**Comment by hwDCN** [comment]
+1

**Review by yuandongx — APPROVED** [review]
LGTM

**Comment by xuxiaowei0512** [comment]
shipit

**Pull request merged**